profiles/seccomp/seccomp_default.go

```diff
@@ -1620,6 +1623,26 @@ func DefaultProfile(rs *specs.Spec) *types.Seccomp {
 				Args:   []*types.Arg{},
 			},
 		}...)
+	case "s390", "s390x":
+		syscalls = append(syscalls, []*types.Syscall{
+			{
+				Name:   "s390_runtime_instr",
```

Comment:
minor nit I was trying to keep the syscalls in alphabetical order everywhere in this file (not 100% sure they all are but I just noticed here...)